morib/src/main/java/org/morib/server/api/allowGroupView/controller/AllowedGroupViewController.java

```diff
@@ -0,0 +1,37 @@
+package org.morib.server.api.allowGroupView.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.morib.server.api.allowGroupView.dto.AddAllowSiteInAllowGroupRequestDto;
+import org.morib.server.api.allowGroupView.facade.AllowedGroupViewFacade;
+import org.morib.server.global.common.BaseResponse;
+import org.morib.server.global.message.SuccessMessage;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v2/allowServiceSet")
```

Comment:
아 혹시! allowedServiceSet 으로 변경하는 건 어떨까요? 다른 허용 ~ 이런 아이들이 다 과거형으로 적혀있어서요!

```suggestion
@RequestMapping("/api/v2/allowedServiceSet")
```

Reply:
넵! 좋습니다!